src/explore-education-statistics-admin/src/pages/release/footnotes/components/FootnoteForm.tsx

```diff
@@ -35,10 +35,25 @@ const FootnoteForm = ({
 }: Props) => {
   const initialValues = useMemo<BaseFootnote>(() => {
     const subjects = mapValues(footnoteMeta.subjects, subject => {
-      const { indicators, filters } = subject;
+      const { indicators, filters, subjectId } = subject;
+
+      let subjectSelectionType = 'NA';
+
+      if (footnote && Object.keys(footnote.subjects).includes(subjectId)) {
+        const foundSubject = footnote.subjects[subjectId];
+        if (foundSubject.selected) {
+          subjectSelectionType = 'All';
+        } else if (
+          !foundSubject.selected &&
+          (Object.keys(foundSubject.filters).length > 0 ||
+            Object.keys(foundSubject.indicatorGroups).length > 0)
+        ) {
+          subjectSelectionType = 'Specific';
+        }
+      }
 
       return {
-        selected: false,
+        selectionType: subjectSelectionType,
```

Comment:
Might as well just call the variable `selectionType` instead of `subjectSelectionType`?